todoapp/templates/todoapp/update_todo.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'todoapp/base.html' %}
+{% block content %}
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Update Todo</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx"
+      crossorigin="anonymous"
+    />
```

Comment:
if you are extending base.html then you dont' need to use these. You just need to use `content` block.

Reply:
Got it. Mostly i cared about only the functionality.